packages/sdk/src/exports.ts

```diff
@@ -46,15 +46,17 @@ export {
     DEFAULT_ENVIRONMENT_ID,
     type EntryPointDiscovery
 } from './Config'
-export { validKeyTypeValues, type KeyType, DEFAULT_KEY_TYPE, IdentityMapping } from './identity/IdentityMapping'
+
+export { DEFAULT_KEY_TYPE } from './identity/IdentityMapping'
 export { GroupKey as EncryptionKey } from './encryption/GroupKey'
 export type { UpdateEncryptionKeyOptions } from './encryption/LocalGroupKeyStore'
 export type { StreamDefinition } from './types'
 export { formStorageNodeAssignmentStreamId, peerDescriptorTranslator } from './utils/utils'
 export { Identity, type SignerWithProvider } from './identity/Identity'
 export { EthereumKeyPairIdentity } from './identity/EthereumKeyPairIdentity'
-export { MLDSAKeyPairIdentity } from './identity/MLDSAKeyPairIdentity'
 export { EthereumProviderIdentity } from './identity/EthereumProviderIdentity'
+export { MLDSAKeyPairIdentity } from './identity/MLDSAKeyPairIdentity'
+export { ECDSAKeyPairIdentity } from './identity/ECDSAKeyPairIdentity'
```

Comment:
Could also 
`export { KeyPairIdentity } from './identity/KeyPairIdentity'`
so that the class is available in API docs (and usable in applications if someone needs)